src/NServiceBus.AcceptanceTests/Performance/ProcessingOptimizations/When_limiting_concurrency_via_both_api_and_config.cs

```diff
@@ -7,18 +7,19 @@ namespace NServiceBus.AcceptanceTests.Config
     using NServiceBus.AcceptanceTests.FakeTransport;
     using NServiceBus.Config;
     using NUnit.Framework;
+    using ScenarioDescriptors;
 
     public class When_limiting_concurrency_via_both_api_and_config : NServiceBusAcceptanceTest
     {
         [Test]
         public async Task Should_throw()
         {
-            var context = await Scenario.Define<Context>()
+            await Scenario.Define<Context>()
                 .WithEndpoint<ThrottledEndpoint>(b => b.CustomConfig(c => c.LimitMessageProcessingConcurrencyTo(10)))
                 .Done(c => c.EndpointsStarted)
+                .Repeat(r => r.For(Transports.AllAvailable.SingleOrDefault(t => t.Key == "FakeTransport")))
```

Comment:
In this case it seems that the test is written specifically for the the fake transport. I propose that we move it to a `FakeTransport` and exclude the files just like we did for Msmq?

That said perhaps now is time to split the tests up into 2 new projects "core only" and "relevant for downstreams" ?